Documentation/cmdref/cilium-agent.md

```diff
@@ -125,6 +124,7 @@ cilium-agent [flags]
       --enable-runtime-device-detection                         Enable runtime device detection and datapath reconfiguration (experimental)
       --enable-service-topology                                 Enable support for service topology aware hints
       --enable-session-affinity                                 Enable support for service session affinity
+      --enable-socket-lb                                        Enable socket-based LB for E/W traffic
```

Comment:
My long term goal is to completely get rid of k8s specific terminology from the Cilium LB. This is because k8s Service is just one example what Cilium can LB today. Another one is the standalone LB mode in which a user is expected to configure LB via Cilium API.

>  What are we load balancing?

It can be either k8s Service or a user defined service which not related to k8s. The main thing is `E/W` which denotes that it's LB-ing only in-cluster traffic.

Reply:
Given we already have `--bpf-lb-sock-hostns-only` today, can't we name this option `--bpf-lb-sock` to make it consistent? It would live explicitly under --bpf* given it's an 'expert' setting which normally shouldn't need to be touched.

Reply:
Okay I see. I had the impression from the original documentation that this feature specifically is for load balancing egress socket traffic destined to *k8s* services to a backend. 

Seeing how it's arbitrary E/W traffic, the naming seems okay with me.

Reply:
Yep, prefixing with `--bpf-lb` SGTM, just maybe `--bpf-lb-enable-sock`.